influence_benchmark/RL/base_iteration.py

```diff
@@ -108,11 +108,15 @@ def launch(self):
         except Exception as e:
             if self.wandb:
                 end_time = time.time()
-                run_duration = end_time - start_time
+                run_duration = end_time - start_time  # type: ignore
 
                 if run_duration < 300:
                     print("Run failed within 5 minutes. Tagging run as 'trash'...")
-                    wandb_run.tags = wandb_run.tags + ("trash",)
+                    wandb_run.tags = wandb_run.tags + ("trash",)  # type: ignore
+                    # import wandb
```

Comment:
why is this commented?

Reply:
It was a mistake, it's a stub for a potential way to actually delete wandb runs that I didn't want to commit

Reply:
I added a better comment in a later PR